api/endpoints/reminder/reminder_schemas.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, Field, validator
+
+
+class ReminderResponse(BaseModel):
+    """Scheme representing a response for a Reminder."""
+
+    active: bool
+    author: int = Field(alias="author_id")
+    mentions: list[int]
+    content: str
+    expiration: datetime  # ISO-formatted datetime
+    id: int
+    channel_id: int
+    jump_url: str
+
+    @validator("expiration")
+    def parse_expiration(cls, value: datetime) -> str:  # noqa N805
+        """A parser that transforms datetimes into isoformat."""
+        return value.isoformat()
```

Comment:
We should raise a value error if this fails to be formatted.

Reply:
Before it gets passed to the parser, pydantic will make sure that it is a datetime object, so I'm not sure I see where it could fail.